File: CodeEval/easy/062nmodm/solution.c
#include <stdio.h>
int main(int argc, const char * argv[]) {
    FILE *file = fopen(argv[1], "r");
    int x, y;
    while (fscanf(file, "%d,%d", &x,&y) == 2)
        printf("%d\n", x - y * (int) (x / y));
    return 0;
}

File: CodeEval/easy/062nmodm/solution.php
#!/usr/bin/env php
<?php

$fh = fopen($argv[1], "r");
while (($line = fgets($fh)) !== FALSE) {
    list($n, $m) = explode(",", $line, 2);
    echo $n % $m;
    echo "\n";
}
